python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -480,6 +480,56 @@ def __call__(self):
     assert get_deployment_start_time(controller, "A") == start_time
 
 
+def test_downscaling_with_fractional_smoothing_factor(serve_instance):
```

Comment:
Can you also test a case where the fractional smoothing factor causes us to block downscaling more than 1 replica?

e.g., we're at 3 and should go to 1, but can't because of smoothing factor. we should see it drop to 2, then to 1.